docarray/array/storage/elastic/backend.py

```diff
@@ -159,10 +159,32 @@ def _build_client(self):
         client.indices.refresh(index=self._config.index_name)
         return client
 
-    def _send_requests(self, request) -> List[Dict]:
+    def _send_requests(self, request, **kwargs) -> List[Dict]:
         """Send bulk request to Elastic and gather the successful info"""
+        bulk_config = {}
```

Comment:
do not call bulk_config. From our side it should be simply calling kwargs, would not add any semantic meaning

Reply:
noted